### DISCUSSION TITLE
Fooocus on my PC is too slow

### DISCUSSION THREAD

**Discussion by AriannaIrwind** [lllyasviel/Fooocus#3644] [Q&A]
Hi guys,
I have a big problem with Fooocus on my PC.
It takes 1-2 minutes to generate images regardless of how long/short the prompt is and I have no clue what the issue is

Device specs:
Processor Intel(R) Core(TM) Intel i7-12650H
Nvidia RTX 4050 6GB GDDR6
RAM 16GB DDR4 3200MHz
System type 64-bit operating system, x64-based processor
Windows 11

Fooocus it starts with no problem but the generation of images is really slow.
Any idea ? 

Thanks in advance

**Comment by Lola-Ice** [comment]
The specs you left out would be vey helpful.

Model? Settings? Iterations??
